server/mediaserver_test.go

```diff
@@ -358,6 +382,7 @@ func TestGotRTMPStreamHandler(t *testing.T) {
 }
 
 func TestMultiStream(t *testing.T) {
+	core.MaxSessions = 0
```

Comment:
Is this still needed?

Reply:
NVM, I see now. Perhaps add a comment saying it's needed to set unlimited sessions.

Reply:
yes, this test creates 500 streams, so it fails if capped to 10

Reply:
done